src/components/Table/types.ts

```diff
@@ -25,6 +25,7 @@ export type TTableProps = {
   data: any[]
   withSelect?: boolean
   fixedHeader?: { y: number }
+  handleRowClick?: (row: any) => void
```

Comment:
why row type is any? 
can we use RowType?

Reply:
No because this is not a row itself it's an row data and that data can be different for any specific case

Reply:
but in this line of code ` onClick={() => handleRowClick?.(row.original)}` you use row, which type is RowType. You should set RowType `original `type if it exists for use.

Reply:
Original is an data which we are passing to the table and that type can be we different

Reply:
okay